QA/flag_test.md

```
@@ -0,0 +1,107 @@
+# selectフラグでエクスポート
+./envault export --select
+```
+
+期待される結果:
+- パスワード入力後、TUIインターフェースが表示されること
+
+## 問題が発生した場合
+
+フラグが認識されない場合は、以下の情報を収集してください：
+
+1. 使用中のOS情報
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)